### PR TITLE
Diagnostics (Julia)

### PR DESCRIPTION
**Pull request by ProExpertProg**

### CODE REVIEW

File: cpp/Naive.cpp [hunk 1]
```diff
@@ -115,7 +129,7 @@ namespace ahr {
             });
 
             auto bracketAParPhiG2Ne_K = halfBracket(sliceXY(dGM, A_PAR), dPhiNeG2);
-            auto bracketPhiDeUEKPar_K = halfBracket(dPhi, dUEKPar);
+            auto bracketPhiDeUEKPar_K = halfBracket(dUEKPar,dPhi);
```
Comment (ProExpertProg) [inline cpp/Naive.cpp]: Rename variable to accurately reflect bracket order

File: cpp/Naive.cpp [hunk 2]
```diff
@@ -229,7 +245,7 @@ namespace ahr {
                 });
 
                 auto bracketAParPhiG2Ne_K_Loop = halfBracket(sliceXY(dGM_Loop, A_PAR), dPhiNeG2_Loop);
-                auto bracketPhiDeUEKPar_K_Loop = halfBracket(dPhi_Loop, dUEKPar_Loop);
+                auto bracketPhiDeUEKPar_K_Loop = halfBracket(dUEKPar_Loop,dPhi_Loop);
```
Comment (ProExpertProg) [inline cpp/Naive.cpp]: Here as well

File: viriato_julia/REGK.jl [hunk 3]
```diff
@@ -169,6 +194,7 @@ bperp_max = maximum(bperp)
 omega_kaw = omegakaw(bperp_max) # Function omega_kaw--its value is public in the function def
 
 # Calculate CFL fraction/timestep
+# TODO Viriato has terms with dz here, which I'm not sure are actually zero. Make sure!
```
Comment (ProExpertProg) [inline viriato_julia/REGK.jl:197]: Is this TODO still relevant?

File: viriato_julia/REGK.jl [hunk 4]
```diff
@@ -520,21 +546,21 @@ while t <= tmax
...
+        guess = deepcopy(akpar_new) # guess only updated if error is low enough from this iteration. 
     end # end of p loop
 
     if debugging
         print("End of corrector loop \n")
     end
 
     if divergent
-        t += 1
+        #t += 1
```
Comment (ProExpertProg) [inline viriato_julia/REGK.jl]: Remove?

File: viriato_julia/REGK.jl [hunk 5]
```diff
@@ -520,21 +546,21 @@ while t <= tmax
...
         continue # go to next time loop iteration with divergent = true
     end
     
     if repeat
         noinc = true # Tells the timestep update if the timestep is allowed to increase
-        t += 1
+        #t += 1
```
Comment (ProExpertProg) [inline viriato_julia/REGK.jl]: Remove?

File: viriato_julia/REGK.jl (hunk 3 above)
Comment (avelberg) [reply viriato_julia/REGK.jl:197]: Maybe, need to revisit this later, but 90% sure no change needed

**Review by avelberg — APPROVED** [review]

**Pull request merged**